src/DazCli.js

```diff
@@ -11,9 +12,12 @@ class DazCli extends Daz {
     _run(args, input, output, terminal) {
         if (args.length >= 3) {
             const command = args[2]
-    
+
             if (command === '--help') {
-                this.output.write(`${fs.readFileSync(path.join(__dirname, 'help.txt'))}\n`)
+                exec(`sh ${__dirname}/help.sh`,
+                    (err, stdout, stderr) =>
+                    err ? this.output.write(`${fs.readFileSync(path.join(__dirname, 'help.txt'))}\n`) : console.log(`${stdout}`)
```

Comment:
'output' is already the 'stdout' (SEE index.js), but DazCli doesn't have to tie directly to the latter. that's why, for decoupling, stdout is passed as a param named output. so, for testing purposes, and also other appliances, DazCli is adapting to whatever "output" that follows the same "interface" (as stdout already does). so, CHANGE stdout by output and we are done ,)